lua/tardis/screens/sh_destination.lua

```diff
@@ -200,9 +212,17 @@ TARDIS:AddScreen("Destination", {menu=false}, function(self,ext,int,frame,screen
 		function() TARDIS:RemoveLocation(map,index) updatelist() pendingchanges = true end,
 		"No")
 	end
+	function remove:Think()
+		if list:GetSelectedLine() ~= nil then
```

Comment:
Check state before changing it as this may be computationally expensive to keep setting it